homeassistant/components/analytics_insights/config_flow.py

```diff
@@ -129,17 +144,18 @@ async def async_step_init(
         ]
         return self.async_show_form(
             step_id="init",
+            errors=errors,
             data_schema=self.add_suggested_values_to_schema(
                 vol.Schema(
                     {
-                        vol.Required(CONF_TRACKED_INTEGRATIONS): SelectSelector(
+                        vol.Optional(CONF_TRACKED_INTEGRATIONS): SelectSelector(
```

Comment:
Can we create a common function that returns the schema?

Maybe we could also use `cv.has_at_least_one_key` for it

Reply:
Note that `cv.has_at_least_one_key` cannot be serialized.